### PR TITLE
add service to query spatial reference specifications

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Frontend nutzt noch feste Projektionen](https://trello.com/c/wYl0bdqk/153-frontend-nutzt-noch-feste-projektionen)

File: services/src/handlers/spatial_references.rs [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        .and(warp::get())
+        .and(authenticate(ctx))
+        .and_then(get_spatial_reference_specification)
+}
+
+async fn get_spatial_reference_specification(
+    srs_string: String,
+    _session: Session,
+) -> Result<impl warp::Reply, warp::Rejection> {
+    // TODO: get specification from Proj or some other source
```
Comment (jdroenner) [inline services/src/handlers/spatial_references.rs:35]: we need a task / or story for this. We also need a service where we can request proj-strings / wkt for the SPatialReference in the BAckend.

File: services/Cargo.toml [hunk 2]
```diff
@@ -29,6 +29,7 @@ mpart-async = "0.5"
 num-traits = "0.2"
 paste = "1.0"
 postgres-types = { version = "0.2", features = ["derive"], optional = true }
+proj = "0.22"
```
Comment (jdroenner) [inline services/Cargo.toml:32]: you imported proj but never used it.

Comment (michaelmattig) [reply services/Cargo.toml]: 5b7e4b5da45b96e3cdb67d8226bf25f4e01aca9c

File: services/src/handlers/spatial_references.rs (hunk 1 above)
Comment (michaelmattig) [reply services/src/handlers/spatial_references.rs:35]: I added this to the projection story https://trello.com/c/d4RnZiEG

**Pull request merged**